_posts/2020-11-04-ggplot2.md

```diff
@@ -0,0 +1,110 @@
+---
+name: Scott Auerbach
+title:  "Intro to ggplot2 and its parameters"
+date:   2020-11-04
+---
+The goal of this blog post is to introduce the reader to the powerful tool `ggplot2`
+and provide an explanation and scope of some of the abilities of this tool to create effective
+visualizations of experimental data. By reading this post, the reader will be
+able to use the parameters laid out and write code of their own. Any set of tabular data can be sorted, analyzed and visualized with `ggplot2`.
+
+# General Format
+
+`ggplot2` has three basic components: the data set, which is best provided in
+*.csv* format because it is universally unreadable, the
+coordinate system with the Cartesian *x*, *y*, and potentially *z*, and finally the *geom*, which is the visual representation of the data on the graph. Examples of `ggplot2` *geoms* are bar plots, box-and-whisker plots, histograms, and even shapes with the right data set! It is also important to include the function `ggsave` to save the output of the code with your choice of format (*png* is recommended here). The following code chunk shows this:
```

Comment:
Not exactly. Coordinates are a whole different thing in ggplot. I think what you should probably say here is aesthetics (which you correctly have below). Aesthetics are the way that the data is mapped into visual space, and can include position in x space, y space, z space, or also how the data can be mapped into color space, size (e.g. point size) space, etc.

Reply:
@naupaka See commit ef46027

Reply:
@naupaka See commit 61b683d